src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,31 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int ITEM_TO_RETURN = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        String result = "";
+        try (BufferedReader fileReader = new BufferedReader(new FileReader(fileName));) {
+            TreeSet<Character> stringTreeSet = new TreeSet<>();
+            while (fileReader.ready()) {
+                String string = fileReader.readLine().toLowerCase().replaceAll("[^a-zA-Z]", "");
+                for (int i = 0; i < string.length(); i++) {
+                    if (!stringTreeSet.contains(string.charAt(i))) {
+                        stringTreeSet.add(string.charAt(i));
+                    }
+                }
+            }
+            int setSize = stringTreeSet.size();
+            for (int i = 0; i < setSize; i++) {
+                if (i < ITEM_TO_RETURN) {
```

Comment:
@TarasHryts А чому ти не можеш просто пройтися від 0 до 5?
```suggestion
                for (int i = 0; i < ITEM_TO_RETURN; i++) {
```

Reply:
а якщо розмір сета менше ITEM_TO_RETURN буде помилка